gradle.properties

```diff
@@ -0,0 +1,11 @@
+# default Docker image to use for launching applications
```

Comment:
just curious, no real opinion: any particular reason to define this in gradle.properties instead of as variables in gradle.build?

Reply:
I had a vague sense that gradle.properties was the preferred approach. But also no strong opinion on my part, so if you know of a reason one way or the other, happy to change.

Reply:
I do not, was hoping you did :)

Reply:
Hah. okay Gradle docs recommend using a properties file. https://docs.gradle.org/current/userguide/organizing_gradle_projects.html#declare_properties_in_gradle_properties_file